utils/conmon/Makefile

```diff
@@ -1,18 +1,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=conmon
-PKG_VERSION:=2.0.29
+PKG_VERSION:=2.0.30
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://github.com/containers/$(PKG_NAME)/archive/v$(PKG_VERSION)
-PKG_HASH:=eb4d5e157671a61b88786e44094775194e30e1d0ad0b9d50035532ece78dbc28
+PKG_HASH:=4b0a98fbe8a63c42f60edac25c19aa6606caa7b1e4fe7846fc7f7de0b566ba25
 
 PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_DEPENDS:=golang/host libseccomp
```

Comment:
I'm not familiar with this package. But are you sure you just need that at build and not runtime? For example your package crun has that as dependency:
https://github.com/openwrt/packages/blob/master/utils/crun/Makefile#L29

Reply:
Yes. Build bot here failed because of missing libseccomp.h, so I had to add it as a build dependency. libseccomp is not available for arc, and that's why arc is disabled. Better would be to take parts of libseccomp that are required and integrate them to common, but that would possibly need building that system every time libseccomp updates- so better this way. libseccomp is not needed on runtime, just something from header file.